MobiFlight/MidiBoard/MidiOutputDefinition.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace MobiFlight
+{
+    public class MidiOutputDefinition
+    {
+        public string Label;
```

Comment:
Doc comments :)

Reply:
I'll think about which ones really need a comment and are not self explanatory.